### ISSUE TITLE
Virtual joystick node from bundled code

### ISSUE THREAD

**Issue by Armory-Community-Channel** [armory3d/armory#2837]
Armory 3D really needs a joystick node based on the bundled script that would let users make there own virtual joystick and control the location/ size... Here is the edited code for a single joystick. https://forums.armory3d.org/t/solved-virtual-gamepad-haxe-script-update/4694/4?u=armory_3d_community
![g](https://user-images.githubusercontent.com/116981508/227726807-91edc36f-41a1-447e-8009-e7cc419afc02.PNG)

**Comment by Armory-Community-Channel** [comment]
Already possible but need multitouch to make it usable
[CustomJoyStick2.zip](https://github.com/armory3d/armory/files/11091116/CustomJoyStick2.zip)

**Comment by MoritzBrueckner** [comment]
Hi, I'm not sure I can follow. What exactly would be the advantage of a logic node witht the trait's functionality over the bundled trait? Both the bundled trait and the logic node would practially do the same, but you can transform the bundled trait into a Haxe trait that is "local" to your game with one click which is something you can't do for logic nodes unless you create a library.

Is this issue about adding more customization options to the bundled trait instead?

**Comment by rpaladin** [comment]
Perhaps so that logic node users can customize and control the behavior of the bundled script?

If that's is the case, a logic node called something like `Set Trait Properties` would be more logical, I think.

**Comment by MoritzBrueckner** [comment]
> If that's is the case, a logic node called something like Set Bundled Trait Properties would be more logical, I think.

I agree, just for trait properties in general (independent of the trait type). Whether updating the value will change anything is a matter of the trait implementation though, some properties might only be used during initialization for example.
